code/modules/vehicles/ridden.dm

```diff
@@ -63,27 +61,37 @@
 			to_chat(user, "<span class='warning'>[src] has no key inserted!</span>")
 			message_cooldown = world.time + 5 SECONDS
 		return FALSE
-	if(legs_required)
-		if(user.usable_legs < legs_required)
-			if(message_cooldown < world.time)
-				to_chat(user, "<span class='warning'>You can't seem to manage that with[user.usable_legs ? " your leg[user.usable_legs > 1 ? "s" : null]" : "out legs"]...</span>")
-				message_cooldown = world.time + 5 SECONDS
+
+	if(rider_check_flags & REQUIRES_LEGS && HAS_TRAIT(user, TRAIT_INCAPACITATED) || HAS_TRAIT(user, TRAIT_FLOORED))
```

Comment:
Could you separate the incapacitated check into its own flag?
Incapacitation is a broader concept than legs usage.

Reply:
Sure.